diff --git a/node_modules/rn-animated-tabbar/lib/components/bottom-tabs/index.d.ts b/node_modules/rn-animated-tabbar/lib/components/bottom-tabs/index.d.ts
index a5819f2..a9a26f2 100644
--- a/node_modules/rn-animated-tabbar/lib/components/bottom-tabs/index.d.ts
+++ b/node_modules/rn-animated-tabbar/lib/components/bottom-tabs/index.d.ts
@@ -8,6 +8,7 @@ interface Props {
     }[];
     tabBarBackground: string;
     activeTabBackground: string;
+    activeIconBackColor:string;˚
     navigationHandler: (screen: string) => void;
     textColor: string;
 }
diff --git a/node_modules/rn-animated-tabbar/lib/components/bottom-tabs/index.js b/node_modules/rn-animated-tabbar/lib/components/bottom-tabs/index.js
index 238b388..59ce35f 100644
--- a/node_modules/rn-animated-tabbar/lib/components/bottom-tabs/index.js
+++ b/node_modules/rn-animated-tabbar/lib/components/bottom-tabs/index.js
@@ -6,7 +6,7 @@ import Svg, { Path } from 'react-native-svg';
 import { StaticTabbar, tabHeight as height } from '../static-tabbar';
 const { width } = Dimensions.get('window');
 const AnimatedSvg = Animated.createAnimatedComponent(Svg);
-export const BottomTabs = ({ tabsData, tabBarBackground, activeTabBackground, navigationHandler, textColor, }) => {
+export const BottomTabs = ({ tabsData, tabBarBackground, activeTabBackground,activeIconBackColor, navigationHandler, textColor, }) => {
     const tabWidth = width / tabsData.length;
     // @ts-ignore
     const left = shape
@@ -65,6 +65,6 @@ export const BottomTabs = ({ tabsData, tabBarBackground, activeTabBackground, na
                 } },
                 React.createElement(Path, { ...{ d }, fill: tabBarBackground }))),
         React.createElement(View, { style: [StyleSheet.absoluteFill, { paddingTop: 4 }] },
-            React.createElement(StaticTabbar, { ...{ value, tabs }, navigate: (id) => navigationHandler(id), tabBarBackground: tabBarBackground, textColor: textColor }))));
+            React.createElement(StaticTabbar, { ...{ value, tabs }, navigate: (id) => navigationHandler(id), tabBarBackground: activeIconBackColor, textColor: textColor }))));
 };
 //# sourceMappingURL=index.js.map
\ No newline at end of file
